package-lock.json

```diff
@@ -9687,42 +9726,6 @@
         "url": "https://github.com/sponsors/ljharb"
       }
     },
-    "node_modules/got": {
```

Comment:
Any idea why this was removed? The only change in package.json dependencies was adding `common-tags`, which doesn't have any dependencies.

Reply:
I think this happened because, in earlier iterations of this PR, more dependencies were installed. What do you think about reverting `package.json` and `package-lock.json` to previous versions, and then installing `common-tags`?

Reply:
done.